httpie/client.py

```diff
@@ -108,24 +108,35 @@ def finalize_headers(headers):
 
 
 def get_default_headers(args):
-    write_info('get_default_headers', 'Total: 3')
+    write_info('get_default_headers', 'Total: 7')
     default_headers = {
         'User-Agent': DEFAULT_UA
     }
 
     auto_json = args.data and not args.form
     if args.json or auto_json:
```

Comment:
Shouldn't this row be replaced by the added blocks below?

Reply:
No, since the blocks below only check which clause of the or expression that made it true.

Reply:
Yeah, missed this, will approve the PR.